.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,13 @@
 #### Before the PR (what is the issue or what needed to be done)
+
 #### The solution
+
 #### Screenshots (if appropriate)
 
-## Checklist
-* [ ] Connect the issue
-* [ ] Test plan
-* [ ] Copyright header
-* [ ] Code formatting
-* [ ] Documentation
+#### Reviewer Checklist
+* [ ] Added an explanation about the work done
+* [ ] Connected the PR and the issue on Zenhub
+* [ ] Added a test plan to the issue
```

Comment:
Maybe we should modify this to also point out that automated tests should be added  as well if necessary?

Reply:
yes good idea!